apps/meteor/app/lib/server/functions/loadMessageHistory.ts

```diff
@@ -29,7 +30,9 @@ export async function loadMessageHistory({
 		throw new Error('error-invalid-room');
 	}
 
-	const hiddenMessageTypes = getHiddenSystemMessages(room);
+	const hiddenSystemMessages = (await getSettingCached('Hide_System_Messages')) as MessageTypesValues[];
+
+	const hiddenMessageTypes = getHiddenSystemMessages(room, hiddenSystemMessages);
```

Comment:
This doesn't work like the original version

First version when calling `getHiddenSystemMessages` and the room didn't have a sysMes property, it returned the list of hidden system messages, but already parsing the `mute_unmute` one to `user-muted` and `user-unmuted`. 

This one is just returning `mute-unmute`

can you take a look?